docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md

```diff
@@ -46,6 +49,21 @@ POST /_plugins/_ml/connectors/_create
         "content-type": "application/json",
         "x-amz-content-sha256": "required"
       },
+      "request_body": "{\"inputText\": \"${parameters.inputText:-null}\", \"inputImage\": \"${parameters.inputImage:-null}\"}",
+      "pre_process_function": "connector.pre_process.bedrock.multimodal_embedding",
+      "post_process_function": "connector.post_process.bedrock.embedding"
+    }
+  ]
+}
+```
+**Use painless script**
```

Comment:
If building function is way more straightforward than painless script, why they should use this? What's the benefit? If there's any benefit of using painless script we should explain here.

Reply:
User might using an older version OpenSearch, so we need to keep this for those versions.

Reply:
Can we add that information in the blueprint? [If your OS version is 2.15 or older, you need to use painless script]